.editorconfig

```diff
@@ -424,6 +425,9 @@ dotnet_diagnostic.SX1101.severity = error
 dotnet_diagnostic.SX1309.severity = error
 dotnet_diagnostic.SX1623.severity = none
 
+dotnet_diagnostic.IDE0046.severity = none
```

Comment:
Probably worth opening up these disables for discussion.

Just curious justifications for each one?

Reply:
RCS1158: Static member in generic type should use a type parameter
All the Binding Extensions raise this suggestion, if we want to retain this rule I can go back and mask the warnings in the existing code

IDE0046: Use conditional expression for return
Added as many of the remaining code suggestions will break the code if this is actioned.  if we want to retain this rule I can go back and mask the warnings in the existing code

Reply:
RCS1158 don't really care about.

IDE0046 make a `suggestion`